packages/appstore/src/components/transaction-list/transactions-for-a-day/transactions-for-a-day.tsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
```

Comment:
Suppose we have to add tests for new components

Reply:
Added a test file